communication/contributor-comms/README.md

```diff
@@ -28,10 +30,10 @@ To better inform the Kubernetes contributor community and highlight their work t
 
 We are always open to being a home for contributors to Kubernetes whether you code or not. 
 
+To find out what projects we're currently working on and explore what you could do to contribute:
 * Come to one of our [meetings] (Every Friday at 8am PT). See the [calendar] to add it to your own calendar.
-* Join our [mailing-list], and/or reach out to us on the #sig-contribex channel on the Kubernetes slack ([slack.k8s.io](http://slack.k8s.io)).
-
-Let us know you are interested and if you have any questions.
+* Reach out to us on the #sig-contribex channel on the Kubernetes slack ([slack.k8s.io](http://slack.k8s.io)).
```

Comment:
I think we should update channel from sig-contribex to sig-contribex-comms

Reply:
Should be fixed with my latest commit - sorry I lost track of this!